### PR TITLE
Maintenance work

### PR DESCRIPTION
**Pull request by fmoessbauer**
Just CI and release creation cleanup. No functional changes to any of the extension parts.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -31,7 +31,7 @@ endif
 
 PACKAGE_NAME=Linux-Entra-SSO
 
-RELEASE_TAG=$(shell git describe --match "v[0-9].[0-9]*" --dirty)
+RELEASE_TAG ?= $(shell git describe --match "v[0-9].[0-9]*" --dirty)
```
Comment (jan-kiszka) [inline Makefile:34]: Not needed, you can always call `make RELEASE_TAG=...`, and that will be respected. Or what is your use case?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -37,7 +35,8 @@ jobs:
         run: |
           sudo apt-get update && sudo apt-get install -y make git zip
 
-      - run: make package deb
+      - name: build packages
+        run: RELEASE_TAG=${{ github.ref_name }} make package deb
```
Comment (jan-kiszka) [inline .github/workflows/release.yml:39]: `make RELEASE_TAG=...`

**Review by jan-kiszka — APPROVED** [review]
Looks good. How you customize variables in makefiles might also be a matter of taste - up to you if you change or not.

File: Makefile (hunk 1 above)
Comment (fmoessbauer) [reply Makefile:34]: Right. Actually I forgot about this option. Nonetheless I consider this `?=` as an interface specification to tell users that they deliberately can change this option. This separates it from other variables assigned with `=`, which are not intended to be changed form the outside. But yes, it's a matter of taste.

**Pull request merged**